src/components/emails/WelcomeEmail.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Body,
+  Button,
+  Container,
+  Head,
+  Hr,
+  Html,
+  Link,
+  Preview,
+  Section,
+  Text,
+} from "@react-email/components";
+import * as React from "react";
+import Logo from "@/components/Logo/index";
+
+const baseUrl = process.env.VERCEL_URL
+  ? `https://${process.env.VERCEL_URL}`
+  : "";
+
+interface StripeWelcomeEmailProps {
```

Comment:
1. Use a type not an interface unless you have a specific reason to use an interface (there is only 1 or 2, which is not seen here)
2. Remove all references to stripe